From patchwork Mon Jan  4 00:28:34 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Rosen Penev <rosenp@gmail.com>
X-Patchwork-Id: 1421978
Return-Path: 
 <openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org>
X-Original-To: incoming@patchwork.ozlabs.org
Delivered-To: patchwork-incoming@bilbo.ozlabs.org
Authentication-Results: ozlabs.org;
 spf=pass (sender SPF authorized) smtp.mailfrom=lists.openwrt.org
 (client-ip=2001:8b0:10b:1231::1; helo=merlin.infradead.org;
 envelope-from=openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org;
 receiver=<UNKNOWN>)
Authentication-Results: ozlabs.org;
 dmarc=fail (p=none dis=none) header.from=gmail.com
Authentication-Results: ozlabs.org;
	dkim=pass (2048-bit key;
 secure) header.d=lists.infradead.org header.i=@lists.infradead.org
 header.a=rsa-sha256 header.s=merlin.20170209 header.b=UgorVGPz;
	dkim=fail reason="signature verification failed" (2048-bit key;
 unprotected) header.d=gmail.com header.i=@gmail.com header.a=rsa-sha256
 header.s=20161025 header.b=d3VMSAGw;
	dkim-atps=neutral
Received: from merlin.infradead.org (merlin.infradead.org
 [IPv6:2001:8b0:10b:1231::1])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature RSA-PSS (4096 bits) server-digest
 SHA256)
	(No client certificate requested)
	by ozlabs.org (Postfix) with ESMTPS id 4D8Gkk10Klz9sVy
	for <incoming@patchwork.ozlabs.org>; Mon,  4 Jan 2021 11:30:54 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=merlin.20170209; h=Sender:Content-Transfer-Encoding:
	Content-Type:List-Subscribe:List-Help:List-Post:List-Archive:List-Unsubscribe
	:List-Id:MIME-Version:Message-Id:Date:Subject:To:From:Reply-To:Cc:Content-ID:
	Content-Description:Resent-Date:Resent-From:Resent-Sender:Resent-To:Resent-Cc
	:Resent-Message-ID:In-Reply-To:References:List-Owner;
	bh=hmIzIemwoe0IsZVI5YubkJzqT3ysg7bQLw+udqvqvZI=; b=UgorVGPz42zpFJ/CS7TRqY0lkL
	UpQA/icL4QrseB9gtgO8/uu3cB13Dao5EbIj+JVitA3hdZzdmuWi8LoGZftduwsQArhTbOCrI9udv
	284OEGM5Ex2ovUvEbTkr2X+/ydhhatbqDwaml8XPHGAWHnhd8DT4gWR2y0yaTLYKOmealhvSmPG9v
	cN1p5p5vsgR6mzwbHmoTS95BTT8LBUvkdaEYLlMDuZF95QMUa+A/EFxf1rC051cebNoa3rut6KIFS
	vqJPPaby4NyyFumnhu6SyLeaPnmyCFJNwbaWL6+odwR4JgeG0NUxz3gOLFBC97hJxD3Pn2O8JJ0EY
	WJ+sbHDg==;
Received: from localhost ([::1] helo=merlin.infradead.org)
	by merlin.infradead.org with esmtp (Exim 4.92.3 #3 (Red Hat Linux))
	id 1kwDjz-0007bQ-EL; Mon, 04 Jan 2021 00:28:55 +0000
Received: from mail-pl1-x634.google.com ([2607:f8b0:4864:20::634])
 by merlin.infradead.org with esmtps (Exim 4.92.3 #3 (Red Hat Linux))
 id 1kwDji-0007U5-M9
 for openwrt-devel@lists.openwrt.org; Mon, 04 Jan 2021 00:28:39 +0000
Received: by mail-pl1-x634.google.com with SMTP id e2so13476705plt.12
 for <openwrt-devel@lists.openwrt.org>; Sun, 03 Jan 2021 16:28:38 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=2rh9ez5EBR6QxKhDaXZ4ow6E/0xXpM6jcXTWY0+F6ZQ=;
 b=d3VMSAGwt93fCu80b3kK38XJfjGhAfzD1ivnOq2bO+nwhq1SokxAfI9reuZKCpU1Jl
 wuXQ5mnCkowwldSdM53KmvuFCZ8UPYfR7geK0NHeH8E3Vmc9BtABWnlpDyw5Di3zzW9m
 GugkxGaagJOFZDHS2AddYWDrV9FwJ6OWJF1HfOo3OThwep+gmkkWYbP6xwgg72wKoka9
 lCZvLOttW5tshz2vqcnL3aOPBaY06ifOXkyF2kQPdbnvlfZqUAJksp1SLbwuNp/2i9TY
 5i+cVPsMxslF358CETrz7zk33h3Gd1lnVSXYWxsxzXBC3+SbVNp6Fk+naFKQ0+B/NIaP
 hWsA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=2rh9ez5EBR6QxKhDaXZ4ow6E/0xXpM6jcXTWY0+F6ZQ=;
 b=cE3OoiumVdOzDaH7YzhRfbjZ/VwtJU1clbi3hjJIFSVqVxdk/2gyOZT3tgig3KgCqJ
 i/HWHSXsrcRRgEplSOjlUc24SrhYzXUFK+EIEM7VFHKDtPQ27I8JLy07+SXYUgwaGF2o
 JzPPxt4raISV2UZBYrN0UlQUayq+oZLU8+iD0jXQe2a6zjnp8xgKUnfuFTGeSodEqztV
 odMVRdTecj0hteaTGtAyLoCE5V1N77uWi2DIEkLJPRT93gCyqfb78z6AUNFB4hm6lduo
 tbEPSoiS5JMZ5aDdwYntVSYH4iqdGGBjs/fBbWh+rVYbTyKBLtSWCc+7/XD5vJdkdhUB
 bQtQ==
X-Gm-Message-State: AOAM5308Iat+XS/fY8sU/s4uQLg2SEQ4BnDyZhYbH1rbvbclfPSOoSf8
 NceDY3+7ZH7My7/ZaVEdctQ9mVZbhaJ43w==
X-Google-Smtp-Source: 
 ABdhPJyPDxJbwJUuCCvWLYfJHyhBCC6ji06diksW51Xoy39IbOWkIWs5x+VQR4+Tb2d6ZfNOp+e9oQ==
X-Received: by 2002:a17:90b:242:: with SMTP id
 fz2mr27922932pjb.39.1609720116438;
 Sun, 03 Jan 2021 16:28:36 -0800 (PST)
Received: from mangix-trapnet.lan ([2001:470:1f05:79e::a89])
 by smtp.gmail.com with ESMTPSA id z3sm38467194pgs.61.2021.01.03.16.28.35
 for <openwrt-devel@lists.openwrt.org>
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Sun, 03 Jan 2021 16:28:35 -0800 (PST)
From: Rosen Penev <rosenp@gmail.com>
To: openwrt-devel@lists.openwrt.org
Subject: [PATCH] libnftnl: update to 1.1.8
Date: Sun,  3 Jan 2021 16:28:34 -0800
Message-Id: <20210104002834.762857-1-rosenp@gmail.com>
X-Mailer: git-send-email 2.29.2
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20210103_192838_753263_F17283C3 
X-CRM114-Status: UNSURE (   9.83  )
X-CRM114-Notice: Please train this message.
X-Spam-Score: -0.2 (/)
X-Spam-Report: SpamAssassin version 3.4.4 on merlin.infradead.org summary:
 Content analysis details:   (-0.2 points)
 pts rule name              description
 ---- ----------------------
 --------------------------------------------------
 -0.0 RCVD_IN_DNSWL_NONE     RBL: Sender listed at https://www.dnswl.org/,
 no trust [2607:f8b0:4864:20:0:0:0:634 listed in]
 [list.dnswl.org]
 0.0 FREEMAIL_FROM          Sender email is commonly abused enduser mail
 provider [rosenp[at]gmail.com]
 -0.0 SPF_PASS               SPF: sender matches SPF record
 0.0 SPF_HELO_NONE          SPF: HELO does not publish an SPF Record
 0.1 DKIM_SIGNED            Message has a DKIM or DK signature,
 not necessarily
 valid
 -0.1 DKIM_VALID Message has at least one valid DKIM or DK signature
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from
 author's domain
 -0.1 DKIM_VALID_EF          Message has a valid DKIM or DK signature from
 envelope-from domain
X-BeenThere: openwrt-devel@lists.openwrt.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: OpenWrt Development List <openwrt-devel.lists.openwrt.org>
List-Unsubscribe: <https://lists.openwrt.org/mailman/options/openwrt-devel>,
 <mailto:openwrt-devel-request@lists.openwrt.org?subject=unsubscribe>
List-Archive: <http://lists.openwrt.org/pipermail/openwrt-devel/>
List-Post: <mailto:openwrt-devel@lists.openwrt.org>
List-Help: <mailto:openwrt-devel-request@lists.openwrt.org?subject=help>
List-Subscribe: <https://lists.openwrt.org/mailman/listinfo/openwrt-devel>,
 <mailto:openwrt-devel-request@lists.openwrt.org?subject=subscribe>
Sender: "openwrt-devel" <openwrt-devel-bounces@lists.openwrt.org>
Errors-To: 
 openwrt-devel-bounces+incoming=patchwork.ozlabs.org@lists.openwrt.org

Fix license information.

Fix wrong ABI version. The library is versioned as libnftnl.so.11.4.0

Add PKG_BUILD_PARALLEL for faster compilation.

Remove autoreconf as nothing is being patched.

Minor cleanups for consistency between packages.

Signed-off-by: Rosen Penev <rosenp@gmail.com>
---
 package/libs/libnftnl/Makefile | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/package/libs/libnftnl/Makefile b/package/libs/libnftnl/Makefile
index c56fb068ce..198e01ba22 100644
--- a/package/libs/libnftnl/Makefile
+++ b/package/libs/libnftnl/Makefile
@@ -8,18 +8,19 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libnftnl
-PKG_VERSION:=1.1.7
-PKG_RELEASE:=2
+PKG_VERSION:=1.1.8
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=https://netfilter.org/projects/$(PKG_NAME)/files
-PKG_HASH:=20dbc13f11004aea2c9e479cfb90359cb11fe3446c3140811c18e4ec1648ed8f
-PKG_MAINTAINER:=Steven Barth <steven@midlink.org>
-PKG_LICENSE:=GPL-2.0+
+PKG_HASH:=04a3fa5b08b736268f7e65836b9f05d9d5f438181467bee3c76c3c4a4f3ab711
 
-PKG_FIXUP:=autoreconf
+PKG_MAINTAINER:=Steven Barth <steven@midlink.org>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING
 
 PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -31,7 +32,7 @@ define Package/libnftnl
   DEPENDS:=+libmnl
   TITLE:=Low-level netlink library for the nf_tables subsystem
   URL:=http://www.netfilter.org/projects/libnftnl
-  ABI_VERSION:=12
+  ABI_VERSION:=11
 endef
 
 define Package/libnftnl/description
